test/models/ui/StackingSpec.scala

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.ui
+
+import base.BaseSpec
+import models.ocelot._
+import models.ocelot.stanzas._
+
+class StackingSpec extends BaseSpec {
+
+  trait Test {
+    def stackedCo(typ: CalloutType, stack: Boolean) = Callout(typ, Phrase(), Seq.empty, stack)
+    def stackedRowGroup(stack: Boolean): RowGroup = RowGroup(Seq.empty, Seq.empty, stack)
+  }
+
+  "Stacking" should {
```

Comment:
Your tests fit the general pattern of one, or more, single stanzas followed by a series of stacked groups. You could also try the patterns (stacked group, single stanza, stacked group) and (stacked group, single stanza) for completeness.

Reply:
added more tests as suggested